.github/workflows/sycl-linux-run-tests.yml

```diff
@@ -100,11 +100,15 @@ on:
           - '-u 1001 --device=/dev/dri --privileged --cap-add SYS_ADMIN'
           - '-u 1001 --device=/dev/dri --device=/dev/kfd --privileged --cap-add SYS_ADMIN'
       target_devices:
+        description: |
+          For SYCL-CTS use "level_zero:gpu" insted of "ext_oneapi_level_zero:gpu".
+          ONEAPI_DEVICE_SELECTOR doesn't support the second one.
```

Comment:
We should just change everything to use `level_zero:gpu`.

Reply:
Will it work fine for E2E?

Reply:
If not, making it fine should be part of changing everything. I think it will though.

Reply:
Okay, looks like everything works fine

Reply:
I would prefer if that was split into a separate PR...

Reply:
Like one PR for enabling CTS on L0 GPU and one for this renaming?

Reply:
Yes!